src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IUsingStatement.vb

```diff
@@ -117,5 +117,46 @@ End Module
 
             Assert.Null(op.Declaration)
         End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact>
+        Public Sub TestUsingBlock()
+            Dim source = <![CDATA[
+Imports System
+
+Module Program
+    Class C
+        Implements IDisposable
+        Public Sub Dispose() Implements IDisposable.Dispose
+        End Sub
+    End Class
+    Sub Main(args As String())
+        Using c1 As C = New C, c2 As C = New C'BIND:"Using c1 As C = New C, c2 As C = New C"
+            Console.WriteLine(c1)
+        End Using
+    End Sub
+End Module]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IUsingStatement (OperationKind.UsingStatement) (Syntax: 'Using c1 As ... End Using')
+  Declaration: IVariableDeclarationStatement (0 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Using c1 As ... s C = New C')
```

Comment:
>Declaration: IVariableDeclarationStatement (0 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Using c1 As ... s C = New C') [](start = 2, length = 144)

This doesn't look right, it looks like there should be two declarations.

Reply:
opened bug - https://github.com/dotnet/roslyn/issues/22561